tests/test_util.py

```diff
@@ -78,10 +79,10 @@ def test_name_for_languageset(self):
         eq_("", m([]))
         eq_("English", m(["en"]))
         eq_("English", m(["eng"]))
-        eq_(unicode("español", "utf-8"), m(['es']))
-        eq_(unicode("English & español", "utf-8"), m(["eng", "spa"]))
-        eq_(unicode("español & English", "utf-8"), m("spa,eng"))
-        eq_(unicode("español, English, & äæ (Zhōngwén)", "utf-8"), m(["spa","eng","chi"]))
+        eq_(u"español", m(['es']))
+        eq_(u"English & español", m(["eng", "spa"]))
+        eq_(u"español & English", m("spa,eng"))
+        eq_(u"español, English, & Chinese", m(["spa","eng","chi"]))
```

Comment:
I asked Mauricio whether capitalization was appropriate here and he wasn't sure. He did mention that using a slash to separate the languages would make it seem less like a sentence that needs a consistent set of capitalization rules.

Since we currently have no mixed-language lanes like this I think it's okay to land this and come back to it later.